src/styles/blocks/now-on-view.scss

```diff
@@ -0,0 +1,162 @@
+.now-on-view {
+  &__wrapper {
+    display: grid;
+    row-gap: 48px;
+  }
+
+  &__description {
+    margin: 8px 0 48px;
```

Comment:
Use only one direction of margins top or bottom and check width of the block according to design